corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -405,8 +405,11 @@ <h3 class="caption" data-bind="html: header" tabindex="0"></h3>
                           class: 'group-' + $parent.entryId,
                       },
                   "/>
-          <span data-bind="renderMarkdown: $data"></span>
+          <span data-bind="renderMarkdown: choice.value()"></span>
```

Comment:
I'm not quite following this, how is calling `value` different than just using `choice` here?

Reply:
Since this now depends on choices_v2, each element in that array is now a dictionary that looks something like this: `{value: "text", caption_image: "path/to/image", etc}` rather than previously just being an array of strings.

Reply:
Aha, thank you. But in `{value: "text", caption_image: "path/to/image", etc}`, are the strings meant to be observables? or functions?

Reply:
Definitely not functions, but are observables because of the `Container.prototype.fromJS` method. I'm still digesting your comment about mutating `choices` in that function

Reply:
Hey, I was thinking about this PR after yesterday's performance issues. Do you think that the choices **should** be observables? I'd think those attributes are read-only. I wonder if using the "ignore" option for choices and/or choices_v2 would be a good idea.

This is a bigger question than just this PR - looking at code today, I think there's other data that we're turning into observables that doesn't need to be - just curious what you think.